components/automate-cli/cmd/chef-automate/automateClusterCtlUtil.go

```diff
@@ -219,7 +220,7 @@ func bootstrapEnv(dm deployManager, airgapBundlePath string, saas bool) error {
 	if err != nil {
 		return err
 	}
-	err = dm.generateConfig()
+	err = dm.generateConfig("provision")
```

Comment:
Make the changes for it to work with onprem mode. Also make the state (provision/deploy as a enum)